cypress/e2e/environment.cy.js

```diff
@@ -0,0 +1,483 @@
+const icon = "./cypress/fixtures/test-icon.png";
```

Comment:
https://docs.cypress.io/api/commands/fixture#Syntax

I suppose it is best to use the Cypress way of importing ;)